modules/typogrify.py

```diff
@@ -6,6 +6,10 @@
 import typogrify
 
 
+def typogrify_filter(s):
+    return Markup(typogrify.typogrify(s))
```

Comment:
@mirisuzanne Any reason not to just do this automatically as part of the `rst` filter? Is there a time when you want `|typogrify` without `|rst`?

Reply:
smart!